internal/controlplane/server_test.go

```
@@ -102,6 +102,47 @@ func TestServerMarksStaleWorkerDisconnected(t *testing.T) {
+func TestServerDeletesOnlyTerminalJobsWithSubmissionAuthorization(t *testing.T) {
+	server, webServer := newTestHTTPServer(t)
+	defer webServer.Close()
+	jobID, err := server.store.CreateJob(t.Context(), "request", "machinist", "agent", "plan", []config.ResolvedAgent{testAgent("plan", "Plan")})
+	if err != nil {
+		t.Fatal(err)
+	}
+	unauthorized := deleteRequest(t, webServer.URL+"/api/v1/jobs/"+jobID, nil)
+	if unauthorized.StatusCode != http.StatusForbidden {
+		t.Fatalf("unauthorized delete status = %d", unauthorized.StatusCode)
+	}
+	unauthorized.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `unauthorized.Body.Close` is not checked

(errcheck)